src/compiler/interpreter.jl

```diff
@@ -198,6 +176,37 @@ function EnzymeInterpreter(
     else
         InferenceParams(; unoptimize_throw_blocks=false)
     end
+    
+    @static if HAS_INTEGRATED_CACHE
```

Comment:
I was also contemplating that Enzyme for 1.11 could simply not use the integrated cache... 
But we might have removed some interface...